docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Logging_How_To.adoc

```diff
@@ -74,3 +74,17 @@ link:Class_Loading_in_WildFly.html#src-557206_ClassLoadinginWildFly-JBossDeploym
 
 This only works for logging in your deployment. Server logs will
 continue to use the logging subsystem configuration.
+
+[[how-do-i-log4j2]]
+== How do I use my own log4j2 implementation?
+
+If you want to use your own log4j2 implementation, such as log4j-core, then you need to do the following two steps.
+
+1. Disable the adding of the logging dependencies to all your deployments with the
+<<add-logging-api-dependencies, add-logging-api-dependencies>> attribute *OR* exclude the logging subsystem in a
+link:Developer_Guide.html#jboss-deployment-structure-file[jboss-deployment-structure.xml]. The module to be excluded is
+`org.apache.logging.log4j.api`_._
```

Comment:
@jamezp should the last sentence be merged into the previous one, a la

*OR* exclude the `org.apache.logging.log4j.api` in a link:Developer_Guide.html#jboss-deployment-structure-file[jboss-deployment-structure.xml]